.github/CODE_OF_CONDUCT.md

```diff
@@ -59,7 +59,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the [project team.](./project_team.md)
```

Comment:
not sure these changes are necessary, either way they're irrelevant for this PR so please revert them

Reply:
Apologies for that, reverted.